password-checker/README.md

```diff
@@ -25,3 +25,8 @@ cargo run --release
 # And now, some fine print
 
 This repository contains example code meant to illustrate the fundamentals of programming with the zkVM. The password policy (and broader protocol) implemented here is intended for educational purposes only.
+
+
+## Video Tutorial
+
+For a walk-through of this example, check out this [excerpt from our workshop at ZK HACK III](https://www.youtube.com/watch?v=vxqxRiTXGBI&list=PLcPzhUaCxlCgig7ofeARMPwQ8vbuD6hC5&index=5).
```

Comment:
I did a much-simplified password checker example for ZK Hack. It is closely related but it's not really this code. I therefore suggested a rewording.

```suggestion
For a walk-through of the fundamentals of this example, check out this [excerpt from our workshop at ZK HACK III](https://www.youtube.com/watch?v=vxqxRiTXGBI&list=PLcPzhUaCxlCgig7ofeARMPwQ8vbuD6hC5&index=5).
```